### PR TITLE
fix: try to fix KubeSphere 4.1 docs display bugs

### PR DESCRIPTION
**Pull request by ROOMrepair**
refer to #3167

### CODE REVIEW

**Comment by zhuxiujuan28** [comment]
@faweizhao26 先合并吧，再看效果

File: assets/scss/common.scss [hunk 1]
```diff
@@ -295,6 +295,16 @@ section {
   }
 }
 
+.admon{
+  thead > tr > th {
+    padding: 0 10px;
+    margin: 0!important;
```
Comment (donniean) [inline assets/scss/common.scss:301]: You could add a space.

```css
0 !important
```

File: assets/scss/common.scss [hunk 2]
```diff
@@ -295,6 +295,16 @@ section {
...
+    height: 30px;
+    line-height: 30px;
+    color: #ffffff;
```
Comment (donniean) [inline assets/scss/common.scss:304]: Could use abbreviated color values.

```css
color: #fff;
```

File: assets/scss/common.scss [hunk 3]
```diff
@@ -313,6 +332,15 @@ section {
   }
 }
 
+.admon.tip {
+  thead > tr > th {
+    background: #78C578
```
Comment (donniean) [inline assets/scss/common.scss:337]: It is best to unify the case of color values.

**Pull request merged**